src/replicache-options.ts

```diff
@@ -53,6 +53,13 @@ export interface ReplicacheOptions<MD extends MutatorDefs> {
    */
   schemaVersion?: string;
 
+  /**
+   * Previous schema versions of this application's data. This enables recovery
+   * of pending mutations from and garbage collection of data persisted
+   * with one of these schema versions.
+   */
+  previousSchemaVersions?: string[];
```

Comment:
Can we not discover this automatically. Having to provide old schema version is annoying.

Reply:
yeah we just need to store it in either local storage or indexeddb due to lack of support for discovering these on firefox.

Probably indexedb is the way to go as the lack of concurrency control in localstorage makes it tricky to avoid clobbering when trying to add to a map or list.